scripts/docker-dingo-store.sh

```diff
@@ -35,10 +35,10 @@ deploy() {
   echo $COOR_SRV_PEERS | tr ',' '\n' >> ${TMP_COORDINATOR_SERVICES}
```

Comment:
COOR_SRV_PEERS is permanently deleted, so we can just remove this variable here.